Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -848,6 +854,13 @@ private extension SearchViewController {
                                      object: .awesomebarResults,
                                      extras: [key: extra])
     }
+
+    func recordFirefoxSuggestSelectionTelemetry(clickInfo: RustFirefoxSuggestionInteractionInfo, position: Int){
+        TelemetryWrapper.gleanRecordEvent(category: .action, 
+                                          method: .tap,
+                                          object: TelemetryWrapper.EventObject.fxSuggest,
+                                          extras: [TelemetryWrapper.EventValue.fxSuggestionClickInfo.rawValue : clickInfo, TelemetryWrapper.EventValue.fxSuggestionPosition.rawValue : position ])
```

Comment:
```suggestion
                                          extras: [
                                              TelemetryWrapper.EventValue.fxSuggestionClickInfo.rawValue: clickInfo,
                                              TelemetryWrapper.EventValue.fxSuggestionPosition.rawValue: position
                                          ])
```

Style, minor: Swiftlint might also complain about it, but either way, let's break up this long line for readability.